src/features/jobs/JobsView.vue

```diff
@@ -41,10 +41,40 @@
                   }"
                 ></div>
               </v-list-item-action>
+              <v-list-item-action class="ml-1">
+                <v-btn icon v-on:click.prevent="dialog = true">
+                  <v-icon>mdi-delete</v-icon>
+                </v-btn>
+                <v-dialog v-model="dialog" max-width="350">
+                  <v-card>
+                    <v-card-title class="text-h5">
+                      Are you sure you want to delete "{{ job.name }}"?
+                      {{ job.id }}
+                    </v-card-title>
+
+                    <v-card-text>
+                      This item will be deleted immediately. You can't undo this
+                      action.
+                    </v-card-text>
+
+                    <v-card-actions>
+                      <v-spacer></v-spacer>
+
+                      <v-btn color="primary" @click="dialog = false">
+                        Cancel
+                      </v-btn>
+
+                      <v-btn color="primary" text @click="deleteJob(job.id)">
+                        Delete
+                      </v-btn>
+                    </v-card-actions>
+                  </v-card>
+                </v-dialog>
+              </v-list-item-action>
             </v-list-item>
             <v-divider
               v-if="index < Object.keys(jobs).length - 1"
-              :key="job.id"
+              :key="'A' + job.id"
```

Comment:
https://stackoverflow.com/questions/51086657/vue-warn-duplicate-keys-detected-x-this-may-cause-an-update-error